Collision/src/main/java/dk/gruppeseks/bodtrd/collision/CollisionProcessor.java

```diff
@@ -96,7 +102,7 @@ else if (ent.getType() == EntityType.WALL)
             }
             case ENEMY:
             {
-                if (ent.getType() == EntityType.WALL)
+                if (ent.getType() == EntityType.PLAYER || ent.getType() == EntityType.ENEMY || ent.getType() == EntityType.WALL)
```

Comment:
maybe a better solution here is to check the CollisionData value. bounce only happens when colliding with concrete objects.